### PR TITLE
Feature/prime 1566 view email/toa

### PR DESCRIPTION
**Pull request by hogmann**
Had to make a new pr to cover the fact there were migration changes

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-toa-maintenance-view-page/enrollee-toa-maintenance-view-page.component.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    </ng-container>
+  </app-page-subheader>
+
+  <app-page-subheader>
+    <ng-container appPageSubheaderTitle>
+      Preview
+    </ng-container>
+  </app-page-subheader>
+
+  <div [innerHtml]="enrolleeAgreementVersion?.text"></div>
```
Comment (johnnyl6) [inline prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-toa-maintenance-view-page/enrollee-toa-maintenance-view-page.component.html:20]: Note that the TOA template HTML has custom CSS class, which is in `@import 'mixins/agreements';` The final rendering style is slightly different if the css is not imported into `enrollee-toa-maintenance-view-page.component.scss`

**Review by johnnyl6 — APPROVED** [review]
LGTM 🎉

File: prime-dotnet-webapi/Services/EmailRenderingService.cs [hunk 2]
```diff
@@ -28,17 +32,17 @@ public async Task<Email> RenderBusinessLicenceUploadedEmailAsync(string recipien
                 from: PrimeEmail,
                 to: recipientEmail,
                 subject: "Site Business Licence Uploaded",
-                body: await _razorConverterService.RenderTemplateToStringAsync(RazorTemplates.Emails.BusinessLicenceUploaded, viewModel)
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Services/EmailRenderingService.cs:31]: for several of these, the `body:` named attribute is removed, but some are not. Why is that?

Comment (hogmann) [reply prime-dotnet-webapi/Services/EmailRenderingService.cs:31]: That is really weird since i tested these and they were there once
but fixed

**Review by james-hollinger — APPROVED** [review]
LGTM

**Pull request merged**